docs/dev-tools/chopsticks/overview.md

```diff
@@ -207,6 +231,160 @@ npx @acala-network/chopsticks run-block  \
 --block 1000
 ```
 
+??? function "**Output file content**"
```

Comment:
No need for bold here. This should also be a `code` admonition instead of a `function` admonition. I just pushed a commit that updates the styling for `code` admonitions, so if you switch this now, it might look a bit off til you pull in the change

Reply:
I'll also rename the admonitions so they're more clear, but I'll do that sometime in the next couple of days. They started off with these names that were more appropriate at the time, but have evolved